lib/pages/bootstrap/tom_bootstrap_dialog.dart

```diff
@@ -364,35 +377,35 @@ class TomBootstrapDialogState extends State<TomBootstrapDialog>
   }
 
   Future<void> _wipeRecoveryWord() async {
-    await _deleteRecoveryWordsInteractor.execute().then(
-          (either) => either.fold(
-            (failure) {
-              Logs().e(
-                'TomBootstrapDialogState::_wipeRecoveryWord(): wipe recoveryWords failed',
-              );
-              if (Matrix.of(context).twakeSupported) {
-                setState(
-                  () => _uploadRecoveryKeyState =
-                      UploadRecoveryKeyState.wipeRecoveryFailed,
-                );
-              } else {
-                setState(
-                  () =>
-                      _uploadRecoveryKeyState = UploadRecoveryKeyState.initial,
-                );
-              }
-            },
-            (success) => setState(
+    await _deleteRecoveryWordsInteractor.execute().then((either) {
+      _createBootstrap();
+      either.fold(
+        (failure) {
```

Comment:
Pls test case delete key failed also

Reply:
This demo

![signal-2024-07-17-003020_002](https://github.com/user-attachments/assets/81b15f5b-60b6-436e-960b-4ee1ace45d11)